servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -552,4 +587,32 @@ public Completable closeAsyncGracefully() {
             return closeable.closeAsyncGracefully();
         }
     }
+
+    private static final class SdStatusCompletable extends Completable {
+        private static final AtomicReferenceFieldUpdater<SdStatusCompletable, CompletableSource.Processor>
+                processorUpdater = newUpdater(SdStatusCompletable.class, CompletableSource.Processor.class,
+                "processor");
+        private volatile CompletableSource.Processor processor = newCompletableProcessor();
+        private boolean seenError;
```

Comment:
Since this field isn't `volatile`, just add a comment that this is only accessed from `nextError` and `resetError` which are not concurrent.